packages/nodejs/build.sh

```diff
@@ -48,6 +54,66 @@ termux_step_host_build() {
 			--disable-tests
 	fi
 	make -j $TERMUX_PKG_MAKE_PROCESSES install
+
+
+	######
+	# Do a host-build of the LLVM toolchain.
+	# Upstream v8 uses LLVM tooling from the main branch of the LLVM project as
+	# the main branch often contains bug fixes which are not released quickly to
+	# stable releases. Also Ubuntu's LLVM toolchain is too old in comparison to
+	# what Google uses.
+	######
+
+	# The LLVM_COMMIT in use can be found in deps/v8/DEPS file,
+	#
+	# For instance, if the deps/v8/DEPS file contains:
+	#
+	#   'third_party/llvm-build/Release+Asserts': {
+	#  'dep_type': 'gcs',
+	#  'bucket': 'chromium-browser-clang',
+	#  'objects': [
+	#    {
+	#      'object_name': 'Linux_x64/clang-llvmorg-21-init-5118-g52cd27e6-5.tar.xz',
+	#      'sha256sum': '790fcc5b04e96882e8227ba7994161ab945c0e096057fc165a0f71e32a7cb061',
+	#      'size_bytes': 54517328,
+	#      'generation': 1742541959624765,
+	#      'condition': 'host_os == "linux"',
+	#    },
+	#
+	# then the LLVM_COMMIT is 52cd27e6. The g before the hash is not part of the
+	# hash, weird that they decided to include a 'g' for no reason, but 'g' isn't
+	# a part of the hexadecimal characters so anyways.. Also v8 project only
+	# stores the short-hash in the DEPS file, but we are using full hash here for
+	# clarity. The full hash can be obtained by having a full checkout of the
+	# llvm-project locally and then running `git log --format=%H -n 1` in the
```

Comment:
Why not download it from https://commondatastorage.googleapis.com/chromium-browser-clang/Linux_x64/clang-llvmorg-21-init-5118-g52cd27e6-5.tar.xz

Reply:
I initially planned to download it straight from Google's build, but couldn't figure out the download url. Thanks!